test/dlc_tests/ec2/pytorch/training/common_cases.py

```diff
@@ -0,0 +1,249 @@
+import os
+
+from packaging.version import Version
+from packaging.specifiers import SpecifierSet
+
+import test.test_utils.ec2 as ec2_utils
+
+from test.test_utils import (
+    CONTAINER_TESTS_PREFIX,
+    get_framework_and_version_from_tag,
+    get_cuda_version_from_tag,
+)
+from test.test_utils.ec2 import (
+    execute_ec2_training_test,
+    get_ec2_instance_type,
+    get_efa_ec2_instance_type,
+)
+
+# Test functions
+PT_STANDALONE_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "pytorch_tests", "testPyTorchStandalone")
+CURAND_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "testCurand")
+PT_MNIST_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "pytorch_tests", "testPyTorch")
+PT_REGRESSION_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "pytorch_tests", "testPyTorchRegression")
+PT_REGRESSION_CMD_REVISED = os.path.join(
+    CONTAINER_TESTS_PREFIX, "pytorch_tests", "testPyTorchRegressionRevised"
+)
+PT_DGL_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "dgl_tests", "testPyTorchDGL")
+PT_APEX_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "pytorch_tests", "testNVApex")
+PT_AMP_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "pytorch_tests", "testPyTorchAMP")
+PT_AMP_INDUCTOR_CMD = os.path.join(
+    CONTAINER_TESTS_PREFIX, "pytorch_tests", "testPyTorchAMPwithInductor"
+)
+PT_TELEMETRY_CMD = os.path.join(
+    CONTAINER_TESTS_PREFIX, "pytorch_tests", "test_pt_dlc_telemetry_test"
+)
+PT_TORCHAUDIO_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "pytorch_tests", "testTorchaudio")
+PT_TORCHDATA_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "pytorch_tests", "testTorchdata")
+
+# Instance type filters
+PT_EC2_CPU_INSTANCE_TYPE = get_ec2_instance_type(default="c5.9xlarge", processor="cpu")
+
+PT_EC2_GPU_INSTANCE_TYPE_AND_REGION = get_efa_ec2_instance_type(default="g4dn.8xlarge")
```

Comment:
use EFA instance type function, as this returns appropriate region for instance types, whereas our other function does not appear to be doing so